python/lsst/sims/catUtils/mixins/snUniversalRules.py

```diff
@@ -0,0 +1,171 @@
+import numpy as np
```

Comment:
The methods in SNUniverse need more complete docstrings (for example, the input parameters for SNparamDistFromHost are not defined)